pr2eus/robot-interface.l

```diff
@@ -106,6 +107,8 @@
 - use-tf2 : use tf2
 - groupname : ros nodehandle group name
 - namespace : ros nodehandle name space
+- joint-states-topic jst : name for subscribing joint state topic
+- joint-states-queue-size : queue size of joint state topic. if you have to different node publishgin joint_state, better to use 2. default is 1
```

Comment:
thank you

--
◉ Kei Okada


On Wed, May 25, 2016 at 8:50 PM, pazeshun <notifications@github.com> wrote:

> In pr2eus/robot-interface.l
> <https://github.com/jsk-ros-pkg/jsk_pr2eus/pull/229#discussion_r64558383>:
>
> > @@ -106,6 +107,8 @@
> >  - use-tf2 : use tf2
> >  - groupname : ros nodehandle group name
> >  - namespace : ros nodehandle name space
> > +- joint-states-topic jst : name for subscribing joint state topic
> > +- joint-states-queue-size : queue size of joint state topic. if you have to different node publishgin joint_state, better to use 2. default is 1
>
> Here.
> "if you have to different node" -> "If you have two different node"
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/jsk-ros-pkg/jsk_pr2eus/pull/229/files/72d792b0693293bc36b65d3b18cde4493a766e06#r64558383>
>